contracts/MemeBase.sol

```diff
@@ -169,4 +150,70 @@ contract MemeBase is MemeFactory {
 
         return msg.value;
     }
+
+    /// @dev Redemption initialization function.
+    /// @param accounts Original accounts.
+    /// @param amounts Corresponding original amounts (without subtraction for burn).
+    function _redemptionSetup(address[] memory accounts, uint256[] memory amounts) private {
+        require(accounts.length == amounts.length);
+
+        redemptionAddress = address(new Meme("Agent Token", "AGENT", DECIMALS, MIN_TOTAL_SUPPLY));
```

Comment:
```suggestion
        redemptionAddress = address(new Meme("Agent Token II", "AGNT II", DECIMALS, 1_000_000_000 ether));
```

Reply:
The original AGNT has 1bn supply. Then 500m got purged (as no-one could collect).

Reply:
Ah right, I'm getting lost in those zero-s.